src/tutorial/output.md

```diff
@@ -0,0 +1,163 @@
+# Output
+
+## Printing "Hello World"
+
+```rust
+println!("Hello World");
+```
+
+Well, that was easy.
+Great, onto the next topic.
+
+## Using println
+
+You can pretty much print all the things you like
+with the `println!` macro.
+This macro has some pretty amazing capabilities,
+but also a special syntax.
+It expects you to write a string literal as the first parameter,
+that contains placeholders that will be filled in
+by the values of the parameters that follow as further arguments.
+
+For example:
+
+```rust
+let x = 42;
+println!("My lucky number is {}.", x);
+```
+
+will print
+
+```console
+My lucky number is 42.
+```
+
+The curly braces (`{}`) in the string above is one of these placeholders.
+This is the default placeholder type
+that tries to print the given in a human readable way.
+For numbers and strings this works very well,
+but not all types can do that.
+This is why there is also a "debug representation",
+that you can get by filling the braces of the placeholder like this: `{:?}`.
+
+For example,
+
+```rust
+let xs = vec![1, 2, 3];
+println!("The list is: {:?}", xs);
+```
+
+will print
+
+```console
+The list is: [1, 2, 3]
+```
+
+If you want your own data types to be printable for debugging and logging,
+you can in most cases add a `#[derive(Debug)]` above their definition.
+
+<aside>
+
+**Aside:**
+The human readable printing is done using the [`Display`] trait,
```

Comment:
Although "human readable" is a common term, it suggests that the `Debug` trait is not human-readable, perhaps it's a binary blob or something. Perhaps you could describe `Display` as "output for ordinary humans", and `Debug` as "output for programmers and for debugging"?